actionlib_msgs/package.xml

```diff
@@ -4,8 +4,8 @@
   <name>actionlib_msgs</name>
   <version>3.0.0</version>
   <description>A package containing some message definitions used in the implementation of ROS 1 actions.</description>
-  <maintainer email="michael@openrobotics.org">Michael Carroll</maintainer>
-  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
```

Comment:
Same comment as https://github.com/ros2/tinyxml_vendor/pull/25#discussion_r744710444

Reply:
Done in 2d65130.